src/ParticularTemplates/Saga/.template.config/template.json

```diff
@@ -20,6 +20,11 @@
       "replaces": "NamespaceName",
       "defaultValue": "Handlers"
     },
+    "ImplicitUsings": {
+      "type": "bind",
+      "binding": "msbuild:ImplicitUsings",
+      "replaces": "ImplicitUsingsValue"
```

Comment:
Is this needed? The other item template doesn't have this line and I don't see where it's being used.